src/package.json

```diff
@@ -3,21 +3,34 @@
   "version": "1.1.0",
```

Comment:
@libpitt can you update the version to 1.2.0 in both `package.json` and the VERSION file?